tests/confd_kubeconfig

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
```

Comment:
confd does, it is not aware of first checking localhost, and we are not running as a k8s pod so it can't glean the details from the environment.